packages/cubejs-cockroach-driver/README.md

```diff
@@ -0,0 +1,16 @@
+<p align="center"><a href="https://cube.dev"><img src="https://i.imgur.com/zYHXm4o.png" alt="Cube.js" width="300px"></a></p>
+
+[Website](https://cube.dev) • [Docs](https://cube.dev/docs) • [Blog](https://cube.dev/blog) • [Slack](https://slack.cube.dev) • [Discourse](https://forum.cube.dev/) • [Twitter](https://twitter.com/thecubejs)
+
+[![npm version](https://badge.fury.io/js/%40cubejs-backend%2Fserver.svg)](https://badge.fury.io/js/%40cubejs-backend%2Fserver)
+[![GitHub Actions](https://github.com/cube-js/cube.js/workflows/Build/badge.svg)](https://github.com/cube-js/cube.js/actions?query=workflow%3ABuild+branch%3Amaster)
+
+# Cube.js Cockroach Database Driver
+
+Pure Javascript Cockroach driver.
+
+[Learn more](https://github.com/cube-js/cube.js#getting-started)
+
+### License
+
+Cube.js Cockroach driver is [Apache 2.0 licensed](./LICENSE).
```

Comment:
It would be great if you would provide some information about the driver configuration necessary steps (required environment variables etc).